deploy/otel-collector.yaml

```diff
@@ -20,9 +20,21 @@ receivers:
                 include_metadata: true
                 cors:
                     allowed_origins:
-                        - '*'
+                        # Can't use '*' with credentials because it fails CORS
+                        # validation on the preflight request.
```

Comment:
@Vadman97 maybe not relevant here, but thought it made sense to keep the configs identical.